neutromeratio/parameter_gradients.py

```diff
@@ -337,11 +341,10 @@ def compute_free_energy_difference(self):
     print(f"Free energy difference {(deltaF.item() * kT).value_in_unit(unit.kilocalorie_per_mole)} kcal/mol")
     # let's say I had a loss function that wanted the free energy difference
     # estimate to be equal to 6:
```

Comment:
note: comment and code are now out of sync

Reply:
yes, that's true! I think this will need a major comment updating session! I will take a stab at this over the weekend